packages/server/src/manifest-builder.ts

```diff
@@ -28,10 +28,16 @@ function* updateSourceMapURL(filePath: string, sourcemapName: string){
   let onRead = yield on(readStream, 'data');
   let currentURL = yield onRead.next();
   
-  if( currentURL.value.toString('utf-8') == '/manifest.js.map'){
-    yield truncate(filePath, size - 16);
-    fs.appendFileSync(filePath, sourcemapName);
-  };
+  try {
+    if( currentURL.value.toString('utf-8') == '/manifest.js.map' ){
+      yield truncate(filePath, size - 16);
+      fs.appendFileSync(filePath, sourcemapName);
+    } else {
+      throw "Unexpected placement and or formatting of sourceMappingURL in the manifest"
```

Comment:
While JavaScript lets you throw _anything_ It's always a good idea to use an `Error` or subclass of `Error` for the main reason that you get a uniform API for handling errors at any point in the code, including a helpful message and a stack trace.

```suggestion
      throw new Error(`Expected a sourcemapping near the end of the generated test bundle, but found "${currentURL}" instead`);
```

Also, I'm not sure if we should be throwing a hard error, or just a warning. what do you think?

Reply:
I would think a warning would be more appropriate than a hard error because a mismatch in the sourcemapURL would just mess up the reporting aspect of BigTest and not the tests itself, right?

Reply:
Hmmm.... I see that, but on the other hand, it means that something is wrong with the "system" and maybe we should be doing a hard fail.

we can always decide to catch the hard fail higher up and log a warning there. After seeing this in action, let's go with an error. A nice knock-on side-effect of this is that it will make testing the failure condition much eaiser.